src/main/java/org/codeforamerica/messaging/services/MessageService.java

```diff
@@ -61,6 +106,7 @@ public Message saveMessage(MessageRequest messageRequest) {
                 .body(body)
                 .toPhone(messageRequest.getToPhone())
                 .toEmail(messageRequest.getToEmail())
+                .messageBatch(messageBatch)
                 .build();
         messageRepository.save(message);
         return message;
```

Comment:
I know this isn't code changed in this PR, but I believe it would be more correct to say `return messageRepository.save(message)`, since the object may be updated while being saved.

Reply:
Not more correct, but just correct as opposed to wrong! :-)